README.md

```diff
@@ -438,8 +439,8 @@ metric's computation function at a regular interval, and also has provisions
 for sending your metric other metrics, so that you can chain metrics
 together.
 
-The `ticker` is where you should return Messages (Gauge or Derived values) and
-keep track of how 'far long' you've computed, as to avoid returning the same
+The `ticker` is where you return Messages (Gauge or Derived values) to
+keep track of how 'far along' you've reached, in order to avoid returning the same
```

Comment:
Another [interesting change](https://ell.stackexchange.com/questions/11517/to-versus-in-order-to), thanks.